src/pages/GiftAid/utils/Utils.js

```diff
@@ -74,7 +74,7 @@ export const getFormValues = (validation, urlId = null, update = false) => {
     }
     // set Giftaid choice for update form
     if (key === 'giftAidClaimChoice') {
-      fieldValues.confirm = value; // reassign to confirm field
+      fieldValues.confirm = value * 1; // reassign to confirm field
```

Comment:
sometimes it's int, sometimes string, I wanted to make sure it has one type (int now)

Reply:
Out of interest, why `* 1` rather than using `parseInt`?